ui/src/components/home/LifeGroupSection.js

```diff
@@ -26,56 +26,76 @@ const colorfulString = (text, colorArray) => {
 };
 
 const lg = {
-  title: colorfulString("LIFE", ["#FD7B7E", "#43B77B", "#7DABFC", "#FEDD64"]),
-  text: "There is no better way to get a taste of who we are and what we believe in then to check out one of our LIFE Groups. This is an opportunity to experience the life-changing power of Biblical community.",
+  section: 'Community',
+  title: 'LIFE Groups',
+  text: 'There is no better way to get a taste of who we are and what we believe in then to check out one of our LIFE Groups. This is an opportunity to experience the life-changing power of Biblical community with Love, Investment, Faith, and Enjoyment.',
 };
-const buttonText = "LEARN MORE >";
+const buttonText = 'Learn More';
 
 const LifeGroupSection = () => {
   return (
-    <Flex w="full" h="100vh">
+    <Flex w="full" h={['100vh', '75vh']}>
```

Comment:
think this should be 50vh instead of 75 as per the figma

Reply:
<img width="1438" alt="Screenshot 2022-08-20 at 3 09 09 AM" src="https://user-images.githubusercontent.com/26876443/185690266-63c2271a-0a56-488f-a463-2cdc50aea798.png">

50vh looks like this to me. I'm not sure where do you get that value. also is it because of the screen size difference>

Reply:
i did 1920x1080 in browser